### ISSUE TITLE
Some sequences do not display in the multispecies tagger

### ISSUE THREAD

**Issue by agilly** [agilly/sadpaR#10]
> NV02 seq42, NV05 seq42, PL05 seq22, and PL06 seq20.

The images show the loading animation but images never actually get loaded.

**Comment by agilly** [comment]
Confirmed the images exist:
```
r$> d[location=="NV02" & interval==42]
                                                                                    fn                  dt location   ct interval      ctid
1: /mnt/t/CT_Data/sus_scrofa/renamed/NV02/CT03/NV02__CT03__2023-11-22__16-22-24(1).JPG 2023-11-22 16:22:24     NV02 CT03       42 NV02 CT03
2: /mnt/t/CT_Data/sus_scrofa/renamed/NV02/CT03/NV02__CT03__2023-11-22__16-22-26(2).JPG 2023-11-22 16:22:26     NV02 CT03       42 NV02 CT03
3: /mnt/t/CT_Data/sus_scrofa/renamed/NV02/CT03/NV02__CT03__2023-11-22__16-22-28(3).JPG 2023-11-22 16:22:28     NV02 CT03       42 NV02 CT03
4: /mnt/t/CT_Data/sus_scrofa/renamed/NV02/CT03/NV02__CT03__2023-11-22__16-23-01(1).JPG 2023-11-22 16:23:01     NV02 CT03       42 NV02 CT03
5: /mnt/t/CT_Data/sus_scrofa/renamed/NV02/CT03/NV02__CT03__2023-11-22__16-23-03(2).JPG 2023-11-22 16:23:03     NV02 CT03       42 NV02 CT03
6: /mnt/t/CT_Data/sus_scrofa/renamed/NV02/CT03/NV02__CT03__2023-11-22__16-23-04(3).JPG 2023-11-22 16:23:04     NV02 CT03       42 NV02 CT03
```

**Comment by agilly** [comment]
Should be fixed with 25e3ac9. Only affected events with #img<10 due to bad logic.
